src/exodus/tx.h

```diff
@@ -269,6 +269,24 @@ class CMPTransaction
     uint32_t getGroup() const { return group; }
     uint16_t getGroupSize() const { return groupSize; }
     const exodus::SigmaProof *getSpend() const { return spend.get(); }
+    CAmount getMintAmount() const {
+        std::vector<uint8_t> denoms;
+        std::transform(
+            mints.begin(),
+            mints.end(),
+            std::back_inserter(denoms),
+            [] (std::pair<uint8_t, exodus::SigmaPublicKey> mint) -> uint8_t {
+                return mint.first;
+            });
+
+        return SumDenominationsValue(getProperty(), denoms.begin(), denoms.end());
```

Comment:
We can eliminate the temporary `denoms` by using `boost::make_transform_iterator`.